### PR TITLE
Sounds static reference check

### PR DESCRIPTION
**Pull request by FluffyGhoster**
The CI now checks that all sounds are referenced with a static path

Atomized and branched from #18021

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Comment by FluffyGhoster** [comment]
**Merge this after the other PR was merged**

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**
